fpga/include/villas/fpga/ips/zynq.hpp

```diff
@@ -0,0 +1,78 @@
+/** Zynq VFIO connector node
+ *
+ * Author: Pascal Bauer <pascal.bauer@rwth-aachen.de>
```

Comment:
Please remove the `Author:` and add yourself as another `SPDX-FileCopyrightText` line